README.md

```diff
@@ -20,11 +19,18 @@ Zonemaster LDNS
 
 ## Introduction
 
-This module provides a Perl interface to the [ldns library](https://www.nlnetlabs.nl/projects/ldns/) from [NLnet Labs](https://www.nlnetlabs.nl/). This module includes the necessary C code from ldns, so the library does not need to be globally installed. It does, however, rely on a sufficiently recent version of OpenSSL being present.
+This module provides a Perl interface to the [ldns library] from [NLnet Labs].
+This module includes the necessary C code from ldns, so the library does not need
+to be globally installed. It does, however, rely on a sufficiently recent version
```

Comment:
Maybe add something like this? `though a globally installed (compatible) version of ldns may be used`
The --no-internal-ldns feature is rather important to both NLnet labs and to packagers, and I feel the original wording glosses over that more than appropriate.

Reply:
"This module includes the necessary C code from ldns" is not true anymore. -- I have rewritten so you can see what you think.